source/adapters/native_cpu/context.hpp

```diff
@@ -91,9 +91,9 @@ static usm_alloc_info get_alloc_info(void *ptr) {
 } // namespace native_cpu
 
 struct ur_context_handle_t_ : RefCounted {
-  ur_context_handle_t_(ur_device_handle_t_ *phDevices) : _device{phDevices} {}
+  ur_context_handle_t_(ur_device_handle_t_ *phDevices) : device{phDevices} {}
 
-  ur_device_handle_t _device;
+  ur_device_handle_t device;
```

Comment:
This isn't a reserved name, as far as I know.

Reply:
I suppose you're right since it's a class member. I'll update the title